build.fsx

```diff
@@ -664,11 +664,10 @@ Target.create "DotNetCoreIntegrationTests" (fun _ ->
         "Fake_Core_IntegrationTests.TestResults.xml")
 
 Target.create "TemplateIntegrationTests" (fun _ ->
-    let targetDir = srcDir </> "test" </> "Fake.DotNet.Cli.IntegrationTests"
 
     runExpecto
-        targetDir
-        "bin/Release/net6.0/Fake.DotNet.Cli.IntegrationTests.dll"
+        root
+        ("src" </> "test" </> "Fake.DotNet.Cli.IntegrationTests" </> "bin" </> "Release" </> "net6.0" </> "Fake.DotNet.Cli.IntegrationTests.dll")
```

Comment:
I pulled the change out of a version of the .NET 8 branch, so it picked up the change to avoid hard coded path separators as well